examples/run_k_grid.py

```diff
@@ -11,11 +11,16 @@ def test_run_k_grid():
 
     #### Traditional ASE functionality #####
     from data.model_gen import get_example_slab as slab
-    slab = slab(adsorbate=True)
+    slab111 = slab(adsorbate=True)
     #########
     sampling_density = 0.02 # example value of sampling density /Angstrom
-    k_grid = get_k_grid(slab, sampling_density, verbose=True)
-    assert k_grid == (6, 6, 1)
+    k_grid111 = get_k_grid(slab111, sampling_density, verbose=True)
+    assert k_grid111 == (6, 6, 1)
     assert get_k_grid(molecule("CO2"), sampling_density) == None
 
+    # Test with the strict definition of reciprocal lattice parameters
+    k_grid111 = get_k_grid(slab111, sampling_density, verbose=True, simple_reciprocal_space_parameters=False)
+    assert k_grid111 == (7, 7, 1)
```

Comment:
I'm curious why the k-grid comes back differently - is this a consequence of rounding somewhere? What are the float values we get before rounding?

Reply:
Indeed it would be good to quantify % difference between calculated k-grid values prior to the use of `math.ceil()`

Reply:
OK, so does that mean whether the results are commensurate depends on symmetry? So for 100/110, which have orthogonal vectors, the k-grids are the same?

Reply:
Just realized I made a mistake, the comment above is updated. 
Yes, indeed. The k-grids are the same if input cell is orthogonal, such as fcc110 and fcc100. The constant factor would be 1 / cos(0), i.e. 1.